staff/jaime/taskApi-Client_Server_Alex&Jaime/tasks-api-client/views/form.pug

```diff
@@ -0,0 +1,24 @@
+html
+ head
+ body
+  form(action="/add", method="post")
+    input(type="text", name="task", placeholder="Write the task")
+    input(type="submit" value="ADD")
+
+  label #{message}
+
```

Comment:
You could add a filter to show the title just in case you have some tasks in it.